cdap-app-templates/cdap-etl/cdap-data-pipeline/src/test/java/co/cask/cdap/datapipeline/DataPipelineTest.java

```diff
@@ -396,6 +399,457 @@ private void testErrorTransform(Engine engine) throws Exception {
   }
 
   @Test
+  public void testIntToStringConversion() throws Exception {
+    Gson GSON = new GsonBuilder()
+      .registerTypeAdapterFactory(new CaseInsensitiveEnumTypeAdapterFactory())
+      .create();
+
+    String configJsonNoQuote =
```

Comment:
Why need such a big hand written json to test the string conversion?? Firstly, you should use library instead of hand written json string (e.g. construct one using `JsonElement` from Gson), otherwise it is very difficult to maintain. Secondly, this is unnecessarily big. You should use a small json to test the functionality.